.github/workflows/main.yaml

```
@@ -0,0 +1,94 @@
+      committer:
+        required: false
+        type: string
+        default: "github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>"
+
+jobs:
+  rclone-upload:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
```

yamllint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)